### PR TITLE
Automation fixes

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Google Inc. All Rights Reserved.
+// Copyright 2017 Google Inc. All Rights Reserved.
```
Comment (codingllama) [inline client/client.go]: Some files are 2016, some are 2017. Should we do "Copyright 2016-2017" instead?

Comment (daviddrysdale) [inline client/client.go]: I believe the preference is for a single year, the year of file creation.

Comment (codingllama) [inline client/client.go]: Ack.

File: scripts/presubmit.sh [hunk 2]
```diff
@@ -12,7 +12,10 @@ find . -name "mock_*" -delete
 golint --set_exit_status ./...
 go vet ./...
 misspell -error -locale US .
-find . -name "*.go" | xargs grep -L "Apache License"
+nolicense=$(find . -name "*.go" | xargs grep -L "Apache License")
+if [[ "${nolicense}" ]]; then
+    exit 1
```
Comment (codingllama) [inline scripts/presubmit.sh]: Add echo "${nolicense}", so we can see the list?

Comment (daviddrysdale) [inline scripts/presubmit.sh]: Done.

**Pull request merged**